region/brinstar/green.json

```diff
@@ -739,8 +767,30 @@
           "to": [
             {
               "id": 1,
-              "requires": [ "canPassBombPassages" ],
-              "unlock": null
+              "requires": [
+                "Morph",
+                {"obstacle":{
+                  "id": "A",
+                  "requires":[
+                    {"or":[
+                      "Bombs",
+                      {"and":[
+                        "PowerBomb",
+                        {"cost": [
+                          {"ammo": [
+                            {
+                              "type": "PowerBomb",
+                              "count": 1
+                            }
+                          ]}
+                        ]}
+                      ]}
+                    ]}
+                  ]
+                }}
+              ],
+              "unlock": null,
+              "note": "A speedball strat can be doable here, depending on adjacent runway, but between the grey door and the number of runway tiles in this room that can't be used for that, it might be better to omit it."
```

Comment:
Legit hell to perform. I'd rather omit it.

Reply:
Well, with a long runway on the other side it's probably quite trivial. If not for the grey door, there'd be some pretty easy setups for it. That we're not able to represent right now.

Reply:
I forgot you can go back to kill the sidehopper after breaking the block. 180 on that one